client/pages/update-credentials.vue

```diff
@@ -43,54 +49,57 @@
       />
 
       <!-- Submit Button -->
-      <v-button class="mx-auto" :loading="form.busy || loading">
+      <v-button
```

Comment:
We're migrating away from VButton in favor of [UButton ](https://ui.nuxt.com/components/button)